devito/ir/iet/analysis.py

```diff
@@ -87,25 +87,21 @@ def mark_iteration_parallel(analysis):
             dims = flatten(dims)
 
             # The i-th Iteration is PARALLEL if for all dependences (d_1, ..., d_n):
-            # test0 := (d_1, ..., d_{i-1}) > 0, OR
-            # test1 := (d_1, ..., d_i) = 0
+            # test0 := (d_1, ..., d_i) = 0, OR
+            # test1 := (d_1, ..., d_{i-1}) > 0, OR
```

Comment:
Out of curiosity, were these reversed for performance?
(also, nit: the additional OR at the end of the second comment makes it look like a comment got lost!)